constants/Backend.js

```diff
@@ -113,7 +113,13 @@ export const api = {
     },
     inflightOrders: `${apiRoot}/orders/inflight`,
     getGlobalOrderOffers: `${apiRoot}/offers/globalOrderOffers`,
-    getordersByDateRange: `${apiRoot}/orders`
+    getordersByDateRange: `${apiRoot}/orders`,
```

Comment:
I think what you should do is just use the same getordersByDateRange but provide 3 parameters:
(dateRange, fromDate, toDate) => {
`${apiRoot}/orders?dateRange=${'dateRange'}&fromDate=${fromDate}&toDate=${toDate}`
}

Reply:
k lll try